### DISCUSSION TITLE
precompiled files?

### DISCUSSION THREAD

**Discussion by arnjan** [tbnobody/OpenDTU#710] [Q&A]
I am wondering if there is any precompiled file available as described for the esptool upload. There isn t any *.bin file under Action > OPenDTU Build etc...

**Comment by DPO99** [comment]
You have to read the complete instruction:

"The pre-compiled files can be found on the [github page](https://github.com/tbnobody/OpenDTU) in the tab "Actions" and the sub menu "OpenDTU Build". Just choose the latest build from the master branch (search for "master" in the blue font text but click on the white header text!). You need to be logged in with your github account to download the files."

At **THIS** moment -  use this link for a new installation:
https://github.com/tbnobody/OpenDTU/actions/runs/4450074421

Scroll down and download the "opendtu-generic" file and extract it. Now you can finde the 4 pre-compliled files.

BR
Timo

**Reply by einfacharthur** [reply]
die precompiled files sind allerdings "generic", d.h. nur für ESP32 (esp32dev Board) und nicht für ESP32-S3 oder C3 nutzbar

**Comment by arnjan** [comment]
Many thanks
I need this for a friend living out of proper mobile range.

Am 17. März 2023 19:27:03 UTC schrieb DPO99 ***@***.***>:
>You have to read the complete instruction:
>
>"The pre-compiled files can be found on the [github page](https://github.com/tbnobody/OpenDTU) in the tab "Actions" and the sub menu "OpenDTU Build". Just choose the latest build from the master branch (search for "master" in the blue font text but click on the white header text!). You need to be logged in with your github account to download the files."
>
>At **THIS** moment -  use this link for a new instalation:
>https://github.com/tbnobody/OpenDTU/actions/runs/4450074421
>
>Scroll down and download the "opendtu-generic" file and extract it. Now you can finde the 4 pre-compliled files.
>
>BR
>Timo
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/tbnobody/OpenDTU/discussions/710#discussioncomment-5349293
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Reply by DPO99** [reply]
Pleased Note that you need a Internet connection for the correct time synch! I dont know if there is a possibility to use a offline ntp Server.

Take a look at issues:
OpenDTU needs access to a working NTP server to get the current date & time.

BR
Timo

**Reply by einfacharthur** [reply]
die Zeit lässt sich über den Browser, lokal und offline synchronisieren.
